src/common/packager.ts

```diff
@@ -190,13 +192,42 @@ export class Packager {
             });
     }
 
-    public prewarmBundleCache(platform: string) {
+    public prewarmBundleCache(platform: string): Q.Promise<void> {
         if (platform === "exponent") {
-            return Q.resolve<void>(void 0);
+            return Q.resolve(void 0);
         }
+
         return this.isRunning()
             .then(running => {
-                return running ? this.prewarmBundleCacheWithBundleFilename(`index.${platform}`, platform) : void 0;
+                if (!running) {
+                    return void 0;
+                }
+                const defaultIndex = path.resolve(this.projectPath, "index.js");
+                const oldIndex = path.resolve(this.projectPath, `index.${platform}.js`); // react-native < 0.49.0
+
+                return Q.all([Packager.fs.exists(defaultIndex), Packager.fs.exists(oldIndex)])
+                .then((exists) => {
+                    let bundleName = "";
+                    if (exists[0]) {
+                        bundleName = "index.bundle";
+                    } else if (exists[1]) {
+                        bundleName = `index.${platform}.bundle`;
+                    } else {
+                        this.logger.info(`Entry point at index.js and index.${platform}.js doesn't exist. Skip prewarming...`);
```

Comment:
I believe, that message could be renamed to `None of possible entry points index.js or index.${platform}.js exist. Skip prewarming...`

Reply:
The same meaning as already use. Why we should change this now?